### ISSUE TITLE
Not working with current Paper versions

### ISSUE THREAD

**Issue by Black-Hole** [SBPrime/AsyncWorldEdit#444]
It's not working because injecting a fake plugin into the plugin system is not supported:
https://github.com/PaperMC/Paper/issues/8870

**Comment by Spaex** [comment]
In case SBPrime needs the full error message on startup:

https://pastebin.com/cDm3XcLZ

**Comment by PovilasLT** [comment]
sad that plugin is dead.

**Comment by gmfamily** [comment]
Indeed, the problem occurs from this [Paper pull request](https://github.com/PaperMC/Paper/pull/8108).
Obviously (a joke of the universe), the last build of Paper supporting AWE is the build # 404.
Builds # 405 and later cannot load AWE.

**Comment by PovilasLT** [comment]
@SBPrime pls fix this, we paid for premium version and plugin isn't working.

**Comment by MagmaSlime123** [comment]
Remove if dead.
